client/src/components/DataFiles/DataFilesModals/DataFilesProjectEditDescriptionModal.js

```diff
@@ -64,8 +64,9 @@ const DataFilesProjectEditDescriptionModal = () => {
   const validationSchema = Yup.object().shape({
     title: Yup.string()
       .min(1)
```

Comment:
@duckonomy [CMD](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1626110406145200?thread_ts=1626105294.140000&cid=CQUA4D5KJ) finalized min and max values:
- min title: 3
- max title: 70
- min desc: 0
- max desc: 800

Reply:
@rstijerina Should we wait on Wes's response on HTML validation?

Reply:
We can move forwards using Yup for now